llmfoundry/command_utils/data_prep/convert_text_to_mds.py

```diff
@@ -394,6 +394,13 @@ def convert_text_to_mds(
         reprocess (bool): Whether to always reprocess the given folder of text files
         trust_remote_code (bool): If true, allows custom code to be executed to load the tokenizer
     """
+    # Load the tokenizer once on the main process so that the files are cached to avoid race conditions
+    # in the Hugging Face load code
+    AutoTokenizer.from_pretrained(
```

Comment:
```suggestion
    _ = AutoTokenizer.from_pretrained(
```

maybe this to indicate that we're using it for the side effect and not the return value? Could take or leave this suggestion tbh

Reply:
`_` actually does store it (its just a variable name), so i will leave as is